app/helpers/groups_helper.rb

```diff
@@ -7,4 +7,51 @@ def check_if_system_group?(group, concept_id)
   def group_provider(group)
     check_if_system_group?(group, group['concept_id']) ? 'CMR' : group['provider_id']
   end
+
+  def current_group_provider_acl_admin
```

Comment:
Should end in a '?' because it's returning a boolean?

Reply:
fixed